f5_openstack_agent/lbaasv2/drivers/bigip/refersh_esd.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
```

Comment:
Can we move this file to <repo>/bin?

Reply:
Yes, it will be installed to /usr/bin/ while rpm is installing.
You can refer to the setup.py file.